### ISSUE TITLE
ElementInternals.labels: No screen reader information

### ISSUE THREAD

**Issue by jamieomaguire** [mdn/content#44395]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/ElementInternals/labels

### What specific section or headline is this issue about?

Lack of screen reader information in the examples  section.

### What information was incorrect, unhelpful, or incomplete?

Incomplete - I think it would be useful if this section discussed the expected behaviour of screen reader narrations.

My understanding is that for most browsers, the associated label will be narrated when the interactive control is focused/interacted with. However I am fairly sure that this is not the case on Safari due to [Webkit bug 259124](https://bugs.webkit.org/show_bug.cgi?id=259124) 

Its quite hard to find information on how to properly associate labels with custom elements using ElementInternals in a way that works on screen readers. Examples could be complicated, but even just a caveat/note specifying that additional logic is possibly required could point devs in the right direction.

Cheers!

### What did you expect to see?

Some information either showing how you can realistically wire up associated labels with the custom element for screen readers (as on Safari the form association doesn't appear to be adequate) or just a note that more work may be needed from the developer.

At the very least, outlining what should work out of the box could be useful.

### Do you have any supporting links, references, or citations?

https://bugs.webkit.org/show_bug.cgi?id=259124 - Safari bug for form-associated elements and screen reader narration

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/elementinternals/labels`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/ElementInternals/labels
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/elementinternals/labels/index.md
* Last commit: https://github.com/mdn/content/commit/ce10da0e9d23d241b175d8d68bf93507734b7c48
* Document last modified: 2023-12-08T11:07:32.000Z

</details>
